tensorrt_llm/executor/utils.py

```diff
@@ -41,8 +41,12 @@ def create_mpi_comm_session(
         print_colored_debug(
             f"Using RemoteMpiPoolSessionClient to bind to external MPI processes at {get_spawn_proxy_process_ipc_addr_env()}\n",
             "yellow")
+        hmac_key = os.getenv("TLLM_SPAWN_PROXY_PROCESS_IPC_HMAC_KEY")
+        # Convert the hex string to bytes
+        if hmac_key is not None:
```

Comment:
@Superjomn what will happen if this is None, i.e. the user script is not started with the trtllm-llmapi-launch.

Will this code work? Thx

Reply:
comm_session should always work with `trtllm-llmapi-launch`, when if it is None, it should go to line 54 and doesn't need an IPC.